exporters/otlp/common/build.gradle.kts

```diff
@@ -17,6 +17,7 @@ dependencies {
   api(project(":api:all"))
   api(project(":sdk:all"))
   api(project(":sdk:metrics"))
+  api(project(":sdk-extensions:logging"))
```

Comment:
@jkwatson I think we can make this, and probably all of these SDK dependencies, `compileOnly` so that should avoid a downstream effect

Reply:
```suggestion
  compileOnly(project(":sdk-extensions:logging"))
```